user_docs/en/userGuide.t2t

```diff
@@ -292,6 +295,7 @@ Telephone support includes local numbers in Australia and the USA.
 The [email user groups https://github.com/nvaccess/nvda/wiki/Connect] are a great source of community help, as are [certified NVDA experts https://certification.nvaccess.org/].
 
 You can make bug reports or feature requests via [GitHub https://github.com/nvaccess/nvda/blob/master/projectDocs/issues/readme.md].
+Feel also free to follow the [contribution guidelines https://github.com/nvaccess/nvda/blob/master/.github/CONTRIBUTING.md] if you need some inspiration and you are interested to actively contribute to the community.

```

Comment:
I think this could be rephrased a little better but I'd like to ask @Qchristensen to suggest an alternative

Reply:
Yes I agree.  Perhaps simply:

The [contribution guidelines https://github.com/nvaccess/nvda/blob/master/.github/CONTRIBUTING.md] contain valuable information for contributing to the community.